### ISSUE TITLE
The LG K8 does not operate the mouse wheel

### ISSUE THREAD

**Issue by dejanx** [Genymobile/scrcpy#398]
I play the **Bowling King** game through **scrcpy** (phenomenal app) and I have a problem. My keys are all keys except the dial when I need to play the ball. Then I need to **roll back** the wheel and the ball should drop to go, but it will not. I found this instruction but I do not know if this is for this problem. At this address. How can I fix this? Thank you.
https://github.com/Genymobile/scrcpy/pull/27

**Comment by rom1v** [comment]
I just tried _Bowling King_, I can throw the ball with left click. When do you need the mouse wheel?

**Comment by dejanx** [comment]
Yes, you can throw it but when you hold the left click, drag the ball up and hold the left click to throw the ball / push the mouse forward, but I need to do this. With a left click, I raise the ball and then turn the dial from the bottom of the mouse to the top / turn the wheel back / you understand?
So I need when the ball is raised to turn the wheel and the ball moves, the mouse is motionless. So it's in a blue stacks emulator, but the problem is that I can not control the game at the same time on a PC and phone as here. Because at the first throw of the ball it has to go the right not to move, and second throw on the phone and move the phone to the ball curve.

**Comment by rom1v** [comment]
> With a left click, I raise the ball and then turn the dial from the bottom of the mouse to the top / turn the wheel back / you understand?

Sorry, I think I don't.

When you talk about "mouse wheel", you don't talk about the wheel used to scroll, between the left and the right button?
![675-001](https://user-images.githubusercontent.com/543275/51346243-6fd61700-1a9d-11e9-9463-4881764d0f0a.png)

Anyway, forget about the game for a moment. Go to some Android settings, if you scroll over them with the mouse wheel, do they scroll correctly?

**Comment by dejanx** [comment]
So I'm talking about a point between the left and right buttons. The left and right buttons work normally, but the wheel does not work, it does not react. And with it I throw the ball that goes straight. Here's the picture in which direction I turn the wheel to throw the ball.
![picsart_01-17-11 02 34](https://user-images.githubusercontent.com/4996962/51403427-b9336e80-1b50-11e9-926e-1a6c768dfad6.jpg)

**Comment by rom1v** [comment]
> The left and right buttons work normally, but the wheel does not work, it does not react.

But is the game programmed to react to such a mouse event?

**Comment by dejanx** [comment]
Yes, in the Bluestacks emulator it works, in Air droid works, power mirror works, mirrorWhat radiates the wheel, everywhere in such applications the wheel works. Try it.

**Comment by rom1v** [comment]
OK. I will not be able to test these emulators/mirror apps now (they're all Windows/Mac and proprietary stuff), I'll try to test later somewhere.

When I scroll, I checked that I send the scroll events, they are the same as in list views in general (where it works). I don't know what events the game expects on mouse wheel.

Meanwhile, could you just confirm that mouse scrolling works in list view (like in settings), please?

**Comment by dejanx** [comment]
If you ask me for a mouse to confirm whether it works as a function, I can say that the left button, the right button works, the middle button (dial) - does not work. While holding the left button, lift the ball / release the left button, and then throw the ball with the wheel. Rotate the wheel in the direction of the arrow in the picture (the image is marked with blue / arrow, picture above). The ball must go straight ahead On the bluestacks emulator everything works like I've described. While with Air droid, PowerMirror, mirorrGo when turning the wheel and throwing the ball, the ball goes to the left (this is a mistake, the ball must go straight ahead , as with the bluestacks emulator).

**Comment by rom1v** [comment]
> If you ask me for a mouse to confirm whether it works as a function, I can say that the left button, the right button works, the middle button (dial) - does not work.

What I ask is whether scrolling (mouse wheel) works correctly _oustide_ Bowling King (in the Android Settings for example, when you scroll the list).

**Comment by dejanx** [comment]
Yes, the dial runs in the application menu, I move the wheel (rotate) the application menu goes to the left to the right. Do you mean that?

**Comment by dejanx** [comment]
Rom1v did  you see what is problem?
